test/integration/integration_kubernetes_test.go

```diff
@@ -416,3 +416,86 @@ kubernetes:
 	// https://github.com/kubernetes/kubernetes/pull/110479
 	asserter.droppedTargetCount(t, 1)
 }
+
+func Test_CuratedExperience(t *testing.T) {
```

Comment:
nit for the future, we might change this to something like the k8s filter test to test different small scenarios

Reply:
yes, I though the same, for now it is readable, but it would be better to change it.

Since we are not still 100% sure about the approach I would not invest time in it for now, but you are right!